src/Application/Helpers/SecurityUtil.cs

```diff
@@ -0,0 +1,22 @@
+namespace Application.Helpers;
+
+using System.Security.Cryptography;
+using System.Text;
+
+public static class SecurityUtil
+{
+    public static string Hash(string input)
```

Comment:
Should salt this hash function

Reply:
And salt should be an environment variable.